,col0,col1,col2
0,",,",,
1,"2751, ""Base64.encodeByteBuffer: IOException: {0}"", 0-stack trace of IOException",,
2,"2752, ""Base64: {0}: FileNotFoundException"", 0-name of file",,
3,"2753, ""Base64: {0}: IOException"", 0-name of file",,
4,"2754, ""Base64: {0}: unable to close FileInputStream"", 0-name of file",,
5,"2755, ""Key null for index {0}, count {1}, tablesize {2}"", 0-index",,
6,"2756, ""Error serializing hashtable: {0}"",",,
7,"2757, ""Error deserializing hashtable: {0}"",",,
8,275,", ""{0}: {1}"", 0-directory error message 1-absoulute path",
9,"2759, ""{0}"", 0-error message that came from trying to archive a file",,
10,"2760, ""removeOldestUntil: failed refinding key {0}. index={1}; retry={2}"", 0-Key in LRUHashtable 1-index 2-nextKey",,
11,"2761, ""Failed to replace object"",",,
12,"2762, ""Cannot persist an object because its class is unknown: {0}"", 0-The object that is trying to be written to the stream",,
13,"2763, ""Failed to instance object because of invalid class index {0}."",",,
14,",,",,
15,"2764, ""class {0} or the classes it references not found in classForName. Turn on util/debug to obtain more information."", 0-name of the class",,
16,"2765, ""class {0} not instantiable via ClassUtil.newInstance(): {1}"",",,
17,"2766, ""class {0} not accessable via ClassUtil.newInstance(): {1}"", 0-name of class 1-exception",,
18,"2767, ""UnknownHostException in setupHost: {0}"",",,
19,276,", ""Unexpected UnknownHostException in setupHost: {0}"",",
20,"2769, ""Error loading object: {0}"",",,
21,"2770, ""Failed to copy file.  Source: {0}, Target: {1}, error {2}"", 0-source file name 1-target file name 2-exception",,
22,"2771, ""Error while closing file {0}.  Error {1}."", 0-file name 1-exception",,
23,"2772, ""Invalid protocol for use with FileUtil.file(url): {0}."", 0-urlContext",,
24,"2773, ""Could not create URL for {0}: {1}"", 0-string representation of a URL 1-exception",,
25,"2774, ""Cannot establish base URL {0}: {1}"", 0-filePath 1-exception",,
26,"2776, ""FileReader: IOException while closing streams {0}"", 0-exception",,
27,"2777, ""FileReader couldn''t open stream: {0}"", 0-exception",,
28,277,", ""FileWriter: IOException while closing streams {0}"", 0-exception",
29,"2779, ""FileWriter couldn''t open stream: {0}"", 0-exception",,
30,27,"0, ""HPPollEvent: connection null in processEvent"",",
31,27,"1, ""Exception during shutdown {0}"", 0-exception",
32,27,"2, ""Exception during monitor processing. {0}"", 0-exception",
33,27,"4, ""Exception during installation: {0}"", 0-exception",
34,27,"5, ""Unable to initialize network connectivity on port {0}. Error: {1}"", 0-port number 1-exception",
35,27,"6, ""Unrecoverable error results in death of current thread"",",
36,27,"7, ""ServerUtil.compareFilenames: got exception trying to close files ({0})"", 0-exception",
37,27,,", ""Problem reading Mime Type mapping {0}: {1}"", 0-MimeTypeFile"
38,27,"9, ""{0}"", 0-exception",
39,"2790, ""Exception during service shutdown: {0}"", 0-exception",,
40,"2791, ""While processing service: {0} an exception occurred: {1}"", 0-name of monitor 1-exception",,
41,"2792, ""Chrono {0} started twice."", 0-name",,
42,"2793, ""end of field not followed by newline or comma in """"{0}"""", line {1}"", 0-path 1-lineNumber",,
43,"2794, ""Event loop caught exception : {0}"", 0-e",,
44,"2795, ""Error opening log file {0}, Error: {1}.  Another process or application may have the file open."", 0-file.getAbsolutePath() 1-error",,
45,"2796, ""Unreadable directory {0}"", 0-dirName",,
46,"2797, ""Unable to read contents of directory {0}"", 0-dirName",,
47,279,", ""Unknown hash function for String encountered.\nDefaulting to slower implementation"",",
48,"2799, ""Wait interrupted in lock {0}.  Continuing"", 0-this.name",,
49,2,"00, ""Lock {0} wait time of {1} sec exceeded.  Total wait time: {2} Handler state {3}, iterations{4}"", 0-this.name 1-Integer.toString((int)(conditions.timeoutIntervalMillis()/Date.MillisPerSecond)) 2-Integer.toString(lockHandlerContext.waitTimeSeconds) 3-this 4-Integer.toString(lockHandlerContext.iteration)",
50,2,"01, ""Unable to archive log file: {0} to file {1}.  "", 0-file.getAbsolutePath() 1-saveToFile.getAbsolutePath()",
51,2,"02, ""{0}: purgeTrigger changed from {1} to {2}"", 0-name 1-oldTrigger 2-this.purgeTrigger",
52,2,"03, ""{0}: insufficient purge, growingLRUHashtable"", 0-name",
53,2,"04, ""Error constructing url ({0})"", 0-urlString",
54,2,"05, ""Error reading {0}: {1}"", 0-url 1-e",
55,2,"06, ""Tried to set both the day of the week and the day of the month. Month ignored."",",
56,2,"07, ""Number format problem with {0}"", 0-cursor",
57,2,0,", ""Unknown day of the week, 0-{0}"", 1-dayOfWeekImage"
58,2,"09, ""create temp file: {0} is not a directory"", 0-directory.getAbsolutePath()",
59,2,"10, ""Unable to create temp file in {0} tries"", 0-Integer.toString(TempFileMaxTries)",
60,2,"11, ""Assert failed: {0} \nThread: {1} {2}\n{3}"", 0-message 1-Thread.currentThread().getName() 2-ThreadDebugState 3-stackTrace()",
61,2,"12, ""Unknown File.seperatorChar: {0}"", 0-File.separator",
62,2,"14, ""waitForResults interrupted"",",
63,2,"15, ""queryDone by {0} did not find a waiter for qid {1}"", 0-ThreadNameAbbreviation.getName() 1-qid",
64,2,"45, ""problem loading from {0}: {1}"", 0-url 1-e",
65,2,"46, ""problem loading from {0}: {1}"", 0-source 1-e",
66,2,"47, ""{0} variable is not defined "", 0-name",
67,2,4,", ""{0} variable is not a Procedure "", 0-name"
68,2,"49, ""No Server configured to monitor - terminating..."",",
69,2,"50, ""Unable to launch process {0}.  Error {1}."", 0-this.args[0] 1-Util.stackTrace(e)",
70,2,"51, ""Exception: {0}"", 0-Util.stackTrace(e)",
71,2,"52, ""Exception while trying to read process ({0}), output: {1}"", 0-this 1-Util.stackTrace(e)",
72,2,"53, ""Exception during read from process {0}"", 0-Util.stackTrace(ioe)",
73,2,"54, ""Couldn''t create FileReader: {0}"", 0-fnfe",
74,2,"55, ""Couldn''t create FileReader: {0}"", 0-e",
75,2,"56, ""Unable to launch external sort process {0}"", 0-Util.formatArray(args",
76,2,"57, ""External sort failed to produce output to file {0}. {1}"", 0-this.outputFile.getAbsolutePath() 1-Util.formatArray(command",
77,2,5,", ""Sorted file differs of length of input file.  Input File: {0}, outputfile{1} {2}"", 0-this.inputFile.getAbsolutePath() 1-this.outputFile.getAbsolutePath() 2-Util.formatArray(command"
78,2,"59, ""File {0} does not exist"", 0-file.getAbsolutePath()",
79,2,"60, ""File {0} is not readable"", 0-file.getAbsolutePath()",
80,2,"61, ""Couldn''t create FileWriter: {0}"", 0-fnfe",
81,2,"62, ""Couldn''t create FileWriter: {0}"", 0-e",
82,2,"63, ""HPPollAcceptLoop: Increasing fds size"",",
83,2,"64, ""HPPollAcceptLoop: problem processing nfds = {0}"", 0-nfds",
84,2,"65, ""HPPollEvent: problem writing to loopback: {0}"", 0-ioe",
85,2,"66, ""HPPollReader: caught exception: \n{0}"", 0-Util.stackTrace(e)",
86,2,6,", ""Unable to start server ({0}) and will not be restarted.  Java VM unable to launch server.  Java Error ({1}), {2}"", 0-serverName() 1-standardErrorString 2-Util.formatArray(Arguments"
87,2,"69, ""Server {0}, was intentionally stopped and will not be restarted."", 0-serverName",
88,2,"70, ""Server {0}, exited with an unexpected error code {1} and will not be restarted. Output:({2})"", 0-serverName 1-Integer.toString(exitCode) 2-serverName 4-outputString()",
89,2,"71, ""Server {0}, terminatedandwillberestarted."", 0-pc.serverName()",
90,2,"72, ""Failed to initialize connection for socket {0} because {1}"", 0-socket 1-ioe",
91,2,"73, ""WorkerThreads = {0} running with a minimum of 2"", 0-Util.integer(numWorkerThreads)",
92,2,"74, ""Single user mode: host specified as {0}, but expected {1}"", 0-hostparameter 1-expectedHostName",
93,2,"79, ""{0}"", 0-pe",
94,2,,"0, ""Expected {0} value for parameter {1} but received {2}"", 0-type 1-parameter 2-object"
95,2,,"3, ""Parsing parameter {0}={1} as Integer failed"", 0-parameter 1-string"
96,2,,"4, ""Parsing parameter {0}={1} as Double failed"", 0-parameter 1-string"
97,2,,"5, ""Parsing parameter {0}={1} as Date failed"", 0-parameter 1-string"
98,2,,"6, ""ServerThread.run() caught an exception while running {0}.\n{1}"", 0-target class name 1-Util.stackTrace(t)"
99,2,,"7, ""Tried to map a file but no Mime Type File is loaded."","
100,2,"92, ""Unable to delete temporary file: {0}"", 0-file.getAbsolutePath()",
101,2,"93, ""Adding extra reader thread."",",
102,2,"94, ""Removing extra reader thread."",",
103,2,"95, ""Server config already exists"",",
104,2,"96, ""Server config for server ({0}) does not exist."", 0-serverName",
105,2,"97, ""Thread interrupted: {0}"", 0-Util.stackTrace(ie)",
106,2,9,", ""Could not parse Java system property {0} as an integer: received {1}"", 0-PropertyHTTPSProxyHost 1-port"
107,2,"99, ""error was: {0}={1}"", 0-Integer.toString(result) 1-errormsg",
108,"2900, ""Failure to load any Services for monitoring.  {0}"", 0-Util.stackTrace(e)",,
109,"2901, ""e"",",,
110,"2902, ""Startup failed to sucessfully complete: service {0}"", 0-this.name()",,
111,"2903, ""Service {0} failed to start"", 0-this.name()",,
112,"2904, ""Unhandled exception: {0}"", 0-Util.stackTrace(e)",,
113,"2905, ""Unhandled exception: {0}"", 0-Util.stackTrace(e)",,
114,"2906, ""The current version of the Oracle OCI is not supported by Ariba.  The recommended version is {0}.  The Oracle OCI version is found by mapping the version of file {1}.  The current version of file {2} is {3}.  The expected version is {4}."", 0-minimumVersion.versionImage() 1-SignatureFile 2-SignatureFile 3-this.versionImage() 4-minimumVersion.fileVersion().versionImage()",,
115,"2907, ""Although newer, the current version of the Oracle OCI is not supported by Ariba. The recommended version is {0}.  The Oracle OCI version is found by mapping the version of file {1}.  The current version of file {2} is {3}.  The expected version is {4}."", 0-minimumVersion.versionImage() 1-SignatureFile 2-SignatureFile 3-this.versionImage() 4-minimumVersion.fileVersion().versionImage()",,
116,290,", ""Unhandled exception: {0}"", 0-Util.stackTrace(e)",
117,"2909, ""usage: CSVReaderTest filename.csv, Encoding | NoEncoding, encoding scheme"",",,
118,"2910, ""Expected seed to be a long: {0}"", 0-string",,
119,"2914, ""Previous log file archived as {0}"", 0-lastArchivedFileName",,
120,"2916, ""{0}{1} (total time = {2} ms) ..."", 0-prefix 1-descriptions.firstElement() 2-totalTime/1000",,
121,"2917, ""{0}{1} (total time = {2} us) ..."", 0-prefix 1-descriptions.firstElement() 2-totalTime",,
122,291,", ""{0}{1} ({2} ms)"", 0-prefix 1-description 2-Long.toString(time/1000)",
123,"2919, ""{0}{1} ({2} us)"", 0-prefix 1-description 2-Long.toString(time)",,
124,"2920, ""loadHashtable for {0} failed, got {1}"", 0-url 1-info",,
125,"2921, ""Begin {0}"", 0-location",,
126,"2922, ""End   {0}"", 0-location",,
127,"2924, ""Shutting down all Servers"",",,
128,"2925, ""InterruptedIOException in HPPollAcceptLoop: {0}"", 0-exception",,
129,"2926, ""{0}"", 0-message",,
130,"2927, ""Server {0} failed to launch and will not be restarted."", 0-server name",,
131,292,", ""Server {0} was launched."", 0-server name",
132,"2929, ""Stopping server: {0}"", 0-server name",,
133,"2930, ""Monitoring process: {0}"", 0-server name",,
134,"2933, ""Starting server {0}({1})"", 0-server name 1-node name",,
135,"2934, ""args[{0}] = ({1})"", 0-argument number 1-command line argument",,
136,"2935, ""{0}({1})"", 0-event 1-nodeName",,
137,"2937, ""Unable to find parameter {0}"", 0-parameter",,
138,293,", ""Server Installer - operation: {0}"", 0-operationImage",
139,"2939, ""Server deleted"",",,
140,"2940, ""Server config for server ({0}) does not exist."", 0-server name",,
141,"2941, ""Configuration: {0}"", 0-serverInstallationConfig",,
142,"2942, ""{0}"", 0-ExecutionControlEventShutdown",,
143,"2943, ""Authenticating user={0}, domain={1}"", 0-user 1-domain",,
144,"2944, ""Login was successful."",",,
145,"2945, ""Monitoring service: {0}"", 0-service name",,
146,"2946, ""Stopping service {0} in preparation to start"", 0-server monitor name",,
147,"2947, ""Waiting for service {0} to stop"", 0-server monitor name",,
148,294,", ""Starting service {0}"", 0-server monitor name",
149,"2949, ""Service {0} needs restart"", 0-server monitor name",,
150,"2950, ""Service {0} was stopped intentionally and will not be automatically restarted"", 0-server monitor name",,
151,"2951, ""Service {0} was stopped with an unexpected error code: {1} "", 0-server name 1-exit value",,
152,"2952, ""Service state for service ({0}), unexpected: {1}"", 0-server name 1-ServiceControl.stateImage(state)",,
153,"2953, ""Service {0} now started"", 0-server name",,
154,"2954, ""Stopping service {0}"", 0-server name",,
155,"2955, ""The current version of the Oracle OCI, {0}, is supported by Ariba."", 0-minimum version needed",,
156,"2957, ""GrowingBufferedOutputStream grew to maxsize of {0}"", 0-buf.length",,
157,"2960, ""Problem converting {0}={1} to a URL: {2}"", 0-ParameterConfigURL 1-string 2-e",,
158,"2961, ""The Ariba Server was unable to initialize the log listener named {1} with class name {0}. Please check your config/Parameters.table or listener implementation to make sure that they are correctly configured."", 0-The fully qualified package name of the listener that was trying to be initialized. 1-The name of the subsection in the Parameters.table file",,
159,"2962, ""Null string table returned in Resource Service: {0}"", 0-string table",,
160,"2964, ""test {0}%"", 0-great ARIBA TESTING localization not required.",,
161,"2970, ""Database connection could not be established for: {0}"", 0-jdbcserver",,
162,",,",,
163,39,"0, ""proxyServerSocket : {0}"", 0-Socket proxy server information",
164,39,"1, ""proxySocket : {0}"", 0-Socket proxy information",
165,39,"2, ""socket to remote : {0}"", 0-Remote socket information",
166,"4267, ""The number of threads specified for a worker queue in the parameters table cannot be formatted to an integer. -{0}"", 0-number format exception",,
167,"4277, ""Only one instance allowed for WorkerQueueHome"",",,
168,427,", ""Worker Queues not found in parameters table"",",
169,"4279, ""Queue not initialized for Name {0}"", 0- name of queue",,
170,"4317, ""Chrono {0}: stop() called before start()"", 0-Chrono name",,
171,"4400, ""Interrupt exception happened when mainServerThread is initializing. Message. -{0}"", 0-exception",,
172,"4634, ""Trying to retrieve value of a null key from LRUHashtable: the key should not be null"",",,
173,"4653, ""XMLUtil.getSAXParser() Factory Configuration Error {0}"", 0-exception",,
174,"4654, ""XMLUtil.getSAXParser() Parser Configuration Exception {0}"", 0-exception",,
175,"4655, ""XMLUtil.getSAXParser() SAX Exception {0}"", 0-exception",,
176,"4795, ""Worker Thread has been processing the same event for a very long time: {0}"", 0-ThreadDebugState",,
177,4,"07, ""Worker Thread has been processing the same event for a very long time: {0} : No further information is available because Thread State Debugging has been disabled."", 0-thread",
178,4,"03, ""Class {0} does not extend {1}"", 0-className 1-supposedSuperclassName",
179,4,"46, ""There are {0} events in the queue.\nEvent histogram = {1}\nFirst event of each type = {2}."", 0-Event count\ 1-Event Histogram 2-First event of each type",
180,4,"76, ""Application Initializer bootstrap error. Could not find the DTD cache for {0}"",0-message",
181,4,"77, ""Unable to read the DTD file from the local disk cache. {0}"", 0-exception",
182,4,7,", ""Error while  caching DTD in runtime memory. {0}"", 0-exception"
183,4,"79, ""System Id is corrupted.Version can not be null or empty string.Version: {0}"", 0-versionname",
184,4,,"0, ""Validation should be proper URI, unable to parse this one as URI.URI: {0}"", 0-resource"
185,4,,"1, ""System Id is corrupted.File name can not be determined.Filename got {0}"", 0-Filename"
186,4,,"2, ""Unable to fetch file from cache dir. Cache Growth disabled for {0}"", 0-property"
187,4,,"3, ""Unable to fetch file from cache dir. Insufficient permission for mkdir of {0}"", 0-directory"
188,499,", ""Error loading class {0}: {1}"", 0-className 1-Error",
189,"5245, ""Unable to get application version checksum: {0}"", 0-message",,
190,"5411, ""Initialization complete : {0} Elapsed time: {1} seconds ({2})"", 0-event 1-seconds 2-nodeName",,
191,"6609, ""loadHashtable for {0} failed because this file does not exist"", 0-file",,
192,665,", ""Required parameter {0} is not specified"", 0-parameter name",
193,"6725, ""Security Exception reading the property '{0}': {1}"", 0-property name 1-exception message",,
194,67,"1, ""Restarting Group"",",
195,67,"2, ""Joining running group"",",
196,6,"53, ""For performance reasons, DatabaseLogger does not support Debug severity logging.  Please correct your Logging configuration."",",
197,"7012, ""{0} contains duplicate key entries with key {1}. Using entry on line {2}. Please remove the redundant entry."", 0-csv file 1-key 2-line number",,
198,"7030, ""CompareChecksums - sizes do not match: local checksum size is {0}, while remote checksum size is {1}"", 0-myChecksumsSize 1-otherChecksumsSize",,
199,"7031, ""CompareChecksums - names differ:  local checksum name is {0}, while remote checksum name is {1}"", 0-myElemName 1-otherElemName",,
200,"7032, ""CompareChecksums - values differ for {0}, local checksum is {1}, while remote checksum is {2}"", 0-name 1-myChecksum 2-otherChecksum",,
201,"7036, ""EarlyCompareChecksums - no remote checksums found in cluster"",",,
202,"7037, ""EarlyCompareChecksums - component {0} has checksum mismatch"",",,
203,"7320, ""Invalid Reference {0}"", 0-class",,
204,753,", ""IOException during IOException in ASM Status Check: {0}"", 0-exception message",
205,"7600, ""LogPrinterWriter got {0}. Will retry with {1}"", 0-exception message 1-encoding to retry",,
206,"7611, ""LogPrinterWriter cannot be created with the last attempt using system default encoding {0}. Returning null"", 0-system default encoding",,
207,761,", ""Ignore attempt to set {0} to {1} because {0} already exists as {2}"",  0-field 1-new value 2-current value",
208,"7619, ""AlogAppender.mapToCategory Cannot find the message category {0} from {1}"", 0-category 1-log class name",,
209,"7650, ""The category {0} is not an instance of LogMessageCategoryForLog4j. You will not be able to enable the logging dynamically."", 0-name of Log message category to map to  log4j logger",,
210,"7651, ""Logger {0} is unregistered--no description or PrivateLoggers entry."", 0-logger name",,
211,7,"01, ""List content changed while performing ListUtil.listEquals()"",",
212,,"076, ""Caught an exception while processing XML: {0}\n{1}"", 0-error text 1-stack trace",
213,,"077, ""XMLProcessor failed while processing {0}"", 0-filename",
214,,0,"0, ""XML parsing error at line {0} of {1}: {2}\n{3}"", 0-line number 1-url 2-error message 3-stack trace"
215,,"379, ""Exception during reading line {0} of {1}, perhaps the wrong encoding is used to read the resource"", 0-line number 1-location",
216,,41,", ""safeWriteObject: the file {0} is not writable"", 0-file name"
217,,49,", ""Attempted to delete file ''{0}'' but could not; there is perhaps an existing unclosed stream on this file; scheduling this file for deletion upon server shutdown"", 0-file name"
218,,"499, ""Could not rename file ''{0}'' to ''{1}''"", 0-source file 1-target file",
219,,50,", ""Attempted to delete file ''{0}'' but could not; there is perhaps an existing unclosed stream on this file"", 0-file name"
220,,"510, ""Caught exception trying to close input stream: {0}"", 0-exception",
221,,"511, ""Cannot find resource with id ''{0}'' in string table ''{1}''"", 0-resource/message ID 1-stringTable name",
222,,"512, ""Caught exception trying to close output stream: {0}"", 0-exception",
223,,"515, ""Could not rename file ''{0}'' to ''{1}''; attempting to back up file from ''{2}''"", 0-from file 1-to file 2-backup file",
224,,"777, ""{0} messages are not allowed for the Audit Logger, original message: ''{1}''"", 0-log level 1-original debug message",
225,,77,", ""Audit appenders do not process Strings, string: ''{0}''"", 0-string message"
226,,"779, ""Audit appenders do not process {0} classes"", 0-class name",
227,,7,"0, ""Error committing transaction to database, AuditLoggerMessage is {0}, stack trace: {1}"", 0-auditLoggerMessage 1-stack trace"
228,,7,"5, ""AuditLogger: Too many arguments specified, all but the first {0} elements are ignored in this array: {1}"", 0-number not ignored 1-args array"
229,,7,"6, ""Cannot set DEBUG level for audit logger {0}, use INFO level instead"", 0-audit logger name"
230,,7,"7, ""Unexpected exception logging message id {0} for logger {1}, stack trace: {2}"", 0-messageId 1-logger name 2-stacktrace"
231,,"797, ""Problem sending notification message for audit logger message id {0}, messageId: {1}, mesage: {2}, stackTrace: {3}"", 0-id 1-messageId 2-message 3-stacktrace",
232,,79,", ""AuditNotifier Error: unexpected class type: {0}, no notification sent"", 0-class name"
233,,,"06, ""File ''{0}'' is deleted, but still exists."", 0-file"
234,,,"31, ""Unknown CSV format error code {0} in """"{1}"""", line {2}"", 0-error code 1-path 2-lineNumber"
235,,,"32, ""Unbalanced quotes in """"{0}"""", line {1}"", 0-path 1-lineNumber"
236,,,"96, ""Error while writing to {0}: {1}"", 0-file name 1-stack trace"
237,,"905, ""Error while creating temp directory: {0}"", 0-stack trace",
238,"9066, ""Error while persisting string resources: {0}"", 0-exception",,
239,"9042, ""Attempting to back up file ''{0}'' from ''{1}' failed'"", 0-file to back up 1-file to back up from",,
240,"9043, ""File replacement for file ''{0}'' is in a bad state, backup file is: ''{1}'', temp file is: ''{2}''"", 0-base file 1-back up 2-temp file",,
241,"9157, ""Unable to instantiate the SignalHandler implementation {0}."", 0-class name",,
242,915,", ""Unable to register the SignalHandler {0} for the signal {1} as this handler has already been registered for another signal"", 0-handler 1-signal",
243,"9159, ""Unable to register the SignalHandler {0} for the signal {1}: {2}"", 0-handler 1-signal 2-reason",,
244,"9160, ""Unable to raise the signal {0}: {1}"", 0-signal 1-reason",,
245,"9161, ""Received signal {0}"", 0-signal",,
246,"9316, ""Caught exception while updating the {0} metrics: {1}"", 0-name 1-exception",,
247,9,"91, ""Exception occurred while batch deleting persisted tokens: {1} {2}"", 0-baseids 1-exception stack",
248,102,"0, ""Generated string exceeded max length of {0}"", 0-max length",
249,"10371, ""Error deserializing list: {0}"",",,
250,"10405, ""safeWriteObject: The file {0} does not exists. Creating new file."", 0-file name",,
251,"10410, ""The file {0} will be deleted when the virtual machine terminates."", 0-file name",,
252,10,"43, ""cXML date format does not allow milliseconds. Ignoring milliseconds in a date string = """"{0}"""""", 0-date string",
253,10,,"6, ""Can not create directory """"{0}"""" path=""""{1}"""" stack trace: {2}"", 0 - directory name 1 - directory path 2 - stack trace"
254,10,,"7, ""File """"{0}"""" path=""""{1}"""" is not a directory. Stack trace: {2}"", 0 - directory name 1 - directory path 2 - stack trace"
255,"11772, ""Caught exception while trying to close {0} : {1}"", 0-object to be closed 1-exception",,
256,"11773, ""Exception thrown while getting the content of URL: {0} Exception : {1}"", 0-url 1-exception",,
